src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -86,6 +87,8 @@ public Command parseCommand(String userInput) throws ParseException {
         case SortByPriorityCommand.COMMAND_WORD, SortByPriorityCommand.ALT_COMMAND_WORD:
             return new SortByPriorityCommandParser().parse(arguments);
 
+        case AddReminderCommand.COMMAND_WORD:
+            return new AddReminderCommandParser().parse(arguments);
```

Comment:
Can add the ALT_COMMAND_WORD to the case here, if you want to add.

Reply:
added